lib/node_modules/@stdlib/string/base/truncate-middle-code-point/README.md

```diff
@@ -0,0 +1,92 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2023 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# truncateMiddleCodePoint
+
+> Truncate code points of string in the middle to a specified length.
```

Comment:
Same description change requests as in the code units PR. Namely,

```suggestion
> Truncate code points in the middle of a string in order to return a string having a specified length.
```

Change should be made here and elsewhere.